spec/features/user_creates_account_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
-require_relative '../factories/users'
+require_relative '../factories/factories'
```

Comment:
The typical pattern here is to have the `rails_helper` file load in your dependencies. 

FactoryGirl rails should look in a few places automatically:

> Factories can be defined anywhere, but will be automatically loaded after calling FactoryGirl.find_definitions if factories are defined in files at the following locations:
> 
> test/factories.rb
> spec/factories.rb
> test/factories/*.rb
> spec/factories/*.rb

https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md